templates/telemetry/firefox_kpi_dashboard_v1/query.sql

```diff
@@ -171,27 +248,17 @@ WITH
   -- see https://bugzilla.mozilla.org/show_bug.cgi?id=1552558
   with_imputed AS (
   SELECT
-    *
+    with_forecast.* REPLACE (
+      COALESCE(imputed.mau, with_forecast.mau) AS mau,
+      COALESCE(imputed.mau + imputed.mau * 0.005, with_forecast.mau_high) AS mau_high,
+      COALESCE(imputed.mau - imputed.mau * 0.005, with_forecast.mau_low) AS mau_low )
   FROM
     with_forecast
-  WHERE
-    NOT (`date` BETWEEN '2019-05-15'
-      AND '2019-06-08'
-      AND datasource IN ('desktop_global',
-        'desktop_tier1'))
-  UNION ALL
-  SELECT
-    datasource,
-    'actual' AS type,
-    `date`,
-    mau,
-    -- The confidence interval is chosen here somewhat arbitrarily as 0.5% of
-    -- the value; in any case, we should show a larger interval than we do for
-    -- non-imputed actuals.
-    mau - mau * 0.005 AS mau_low,
-    mau + mau * 0.005 AS mau_high
-  FROM
-    static.firefox_desktop_imputed_mau28_v1 )
+  LEFT JOIN
```

Comment:
The LEFT JOIN and COALESCE here is a more general way of including the imputed values. We have to move to this method since we now are including dau and wau, which aren't present in the imputed table.

Reply:
elegant solution